changelog

```diff
@@ -16,7 +22,7 @@ turnkey-b2evolution-16.0 (1) turnkey; urgency=low
   * Updated version of mysqltuner script - now installed as per upstream
     recommendation.
 
-  * Note: Please refer to turnkey-core's changelog for changes common to all
+  * Note: Please refer to turnkey-core's 16.1 changelog for changes common to all
```

Comment:
It looks like you updated the v16.0 entry, rather than adding the relevant info to your v16.1 entry...

Reply:
Done!